plugins/source/aws/codegen/recipes/organizations.go

```diff
@@ -30,6 +30,15 @@ func OrganizationsResources() []*Resource {
 					},
 				}...),
 		},
+		{
+			SubService:  "organizations",
+			Name:        "aws_organizations",
+			Struct:      &types.Organization{},
+			Description: "https://docs.aws.amazon.com/organizations/latest/APIReference/API_Organization.html",
+			SkipFields:  []string{"AvailablePolicyTypes"}, // deprecated and misleading field according to docs
+			PKColumns:   []string{"arn"},
```

Comment:
We should add the current account ID as a PK as well

Reply:
Otherwise there's no way to go from account to org... makes sense, as `Account` doesn't have any org info.